csunplugged/static/js/fixedsticky.js

```diff
@@ -0,0 +1,193 @@
+;(function( win, $ ) {
```

Comment:
I know this is probably third-party but I feel we should have a comment at the top of scripts or some documentation explaining what functionality each javascript file provides. I always feel that javascript libraries/scripts tend to be poorly named and are not immediately obvious what they are used for let alone how to use them.